HW02-generics/src/main/java/homework/CustomerReverseOrder.java

```diff
@@ -0,0 +1,15 @@
+package homework;
+
+import java.util.LinkedList;
+
+public class CustomerReverseOrder {
+    LinkedList<Customer> customers = new LinkedList<>();
```

Comment:
Поля private.
Тип лучше объявлять минимальным интерфейсом.

```suggestion
    private final Deque<Customer> customers = new LinkedList<>();
```